docs/source/packs.rst

```diff
@@ -235,6 +235,81 @@ than personal access tokens and can be configured on the per-repo basis.
 Other git hosting services should also support either SSH or HTTPS auth,
 and would be configured in a similar fashion.
 
+Installing packs from behind a proxy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you're running |st2| server behind a HTTP/HTTPS proxy, you have to configure |st2| so you
+can install packs from HTTP/HTTPS URLs from behind the proxy. You can setup the proxy
+configuration by setting ``https_proxy`` or ``http_proxy`` environment variables for
+``st2actionrunner`` and ``st2api`` processes. If you are using a HTTPS proxy, depending on
+if you are using
+`CONNECT SSL tunnel <http://wiki.squid-cache.org/Features/HTTPS#CONNECT_tunnel>`__
+or `MITM proxy <http://docs.mitmproxy.org/en/stable/howmitmproxy.html#the-mitm-in-mitmproxy>`__,
+you'll have to optionally pass in path to proxy CA cert (in the MITM case) via the
+``proxy_ca_bundle_path`` environment variable. If you are using Python 2.7 version < ``2.7.9``
+(which is the case in Ubuntu 14.04),
+note that the MITM proxy simply won't work (See
+`CVE-2014-9365 <http://people.canonical.com/~ubuntu-security/cve/2014/CVE-2014-9365.html>`__).
+In those cases, it is recommended to use `CONNECT SSL tunnel <http://wiki.squid-cache.org/Features/HTTPS#CONNECT_tunnel>`__.
+
+Proxy configuration via environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are running Ubuntu, you can edit ``/etc/default/st2actionrunner`` and
+``/etc/default/st2api`` files to set the proxy configuration environment variables.
+If you are using RHEL, you can edit ``/etc/sysconfig/st2actionrunner`` and
+``/etc/sysconfig/st2api`` to place the proxy configuration environment variables for the
+``st2actionrunner`` and ``st2api`` processes. |st2| picks up these environment variables
+to let pack installer know about the proxy. The file contents should look as follows:
```

Comment:
It would be nice to have a note indicating that restarting `st2api` and `st2actionrunner` services is required.